shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/rulealtered/RuleAlteredJobWorker.java

```diff
@@ -163,6 +167,7 @@ private static YamlRootConfiguration getYamlRootConfig(final JobConfiguration jo
      */
     @Subscribe
     public void start(final StartScalingEvent event) {
+        isUncompletedJobOfSameSchemaInJobList(event.getSchemaName());
```

Comment:
`isUncompletedJobOfSameSchemaInJobList` doesn't return result, if this check doesn't pass, it just print log but the process will go on.